packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -3,21 +3,25 @@
 :root {
   --terra-consumer-body-background: url('https://wallpapercave.com/wp/5ynt7Gb.jpg');
   --terra-consumer-nav-active-item-background: var(--terra-consumer-body-background);
+  --terra-consumer-mobile-panel-background: var(--terra-consumer-body-background);
+  --terra-consumer-layout-background: var(--terra-consumer-body-background);
   --terra-consumer-nav-item-color: #fff;
+  --terra-consumer-toggle-header-hover-color: var(--terra-consumer-nav-item-color);
```

Comment:
nit: use the word toggler instead of toggle. Toggle makes me think its a boolean or something dictating if it should toggle